templates/details.tt2

```diff
@@ -21,7 +21,7 @@
   [% uniquetag_one = name | remove('[^A-Za-z0-9]+') %]
   [% uniquetag_two = tag | remove('[^A-Za-z0-9]+') %]
   [% uniquetag = "$uniquetag_one$uniquetag_two" %]
-  <div href="#" class='btn btn-light rounded-0 card-header d-sm-flex justify-content-between align-items-center' data-toggle="collapse" data-target="#collapse[% uniquetag %]" aria-expanded="false" aria-controls="collapse[% uniquetag %]">
+  <div href="#" class='card-header d-sm-flex justify-content-between align-items-center' data-toggle="collapse" data-target="#collapse[% uniquetag %]" aria-expanded="false" aria-controls="collapse[% uniquetag %]">
```

Comment:
Have you checked if this works well with the "suse" theme?

Reply:
It does, in fact it fixes some issues with the theme too

Reply:
Nice!